density-size-correction/density-corrected-image-svg-aspect-ratio.html

```diff
@@ -4,7 +4,6 @@
     <link rel="author" title="Noam Rosenthal" href="noam@webkit.org">
     <link rel="match" href="density-corrected-image-svg-aspect-ratio-ref.html" />
     <meta name="assert" content="Assert that SVG aspect ratio is applied correctly on density-corrected images">
-    <meta name=fuzzy content="10;100">
```

Comment:
Why is it okay to remove the fuzziness now?

Reply:
This data for density-size-correction tests was simply wrong, the difference is much higher, so they'll fail as before, but without exact bogus values.

https://wpt.fyi/results/density-size-correction?label=master&label=experimental&product=chrome&product=webkitgtk&product=firefox&product=safari&aligned&q=density-size-correction